src/clients/kol.ts

```diff
@@ -413,6 +422,32 @@ export class KoLClient {
 
     return match?.[1] ?? null;
   }
-}
 
+  async getPlayerInformation(id: number): Promise<FullPlayer | null> {
+    try {
+      const profile = await this.tryRequestWithLogin("showplayer.php", { who: id });
+      const header = profile.match(/<b>([^>]*?)<\/b> \(#(\d+)\)<br>/);
```

Comment:
consider naming these capture groups instead of using their positioning

Reply:
This would be a neat opportunity to use https://regexp.dev/, something @gausie sent me earlier

Reply:
i am not strongly opposed to this but i am slightly opposed simply by dint of the fact that it would add a random extra dependency for slight future help at the cost of revising every regexp we use in the code right now. the prisma add seemed to be super valuable, this feels... less so, to me. i think current regexp here is relatively easy to read, by regex standards, and it works fine. but i am not strongly opposed, just a mildly salty old man who doesn't like extraneous dependencies that make us beholden to more and more weird random documentation people have to read to understand things.

Reply:
You can use named capture groups without the thing gausie sent me. I'm a big fan of named capture groups. They have names!

Reply:
resolved after discussion re: readability